--- nload-0.4.0/docs/Makefile.am~	Mon Jan  7 10:57:14 2002
+++ nload-0.4.0/docs/Makefile.am	Mon Jan  7 10:57:14 2002
@@ -1,4 +1,4 @@
 ## Helder Correia <helder.correia@netcabo.pt>
 
-## dist_man1_MANS = nload.1.in
-## EXTRA_DIST = $(dist_man1_MANS)
+man_MANS = nload.1
+EXTRA_DIST = nload.1.in
--- nload-0.4.0/Makefile.am~	Mon Jan  7 10:59:11 2002
+++ nload-0.4.0/Makefile.am	Mon Jan  7 10:59:11 2002
@@ -3,11 +3,3 @@
 SUBDIRS = docs src
 DIST_SUBDIRS = $(SUBDIRS)
 EXTRA_DIST = AUTHORS ChangeLog COPYING INSTALL README nload.spec.in
-
-install:
-	$(mkinstalldirs) $(DESTDIR)$(prefix)/man/man1
-	$(INSTALL_DATA) $(top_srcdir)/docs/$(PACKAGE).1 $(DESTDIR)$(prefix)/man/man1
-	gzip $(DESTDIR)$(prefix)/man/man1/$(PACKAGE).1
-
-uninstall:
-	rm $(DESTDIR)$(prefix)/man/man1/$(PACKAGE).1.gz
