diff -burNp nload-0.7.2/src/form_field.h nload-0.7.2-new/src/form_field.h
--- nload-0.7.2/src/form_field.h	2008-02-02 17:14:34.000000000 +0100
+++ nload-0.7.2-new/src/form_field.h	2008-10-11 14:54:03.993388409 +0200
@@ -79,7 +79,7 @@ class Form
                 virtual void slot_fieldChanged(Field* field) {}
         };
         
-        Form(Slots* slots = 0);
+        Form(Slots* slots);
         ~Form();
         
         std::vector<Field *>& fields();
