diff -burNp nload-0.7.2/src/form_field.h nload-0.7.2-new/src/form_field.h
--- nload-0.7.2/src/form_field.h	2008-02-02 17:14:34.000000000 +0100
+++ nload-0.7.2-new/src/form_field.h	2008-10-11 15:11:46.903805424 +0200
@@ -18,7 +18,7 @@
 #ifndef FORM_FIELD_H
 #define FORM_FIELD_H
 
-#include <curses.h>
+#include <ncurses/curses.h>
 #include <form.h>
 #undef clear
 #undef erase
diff -burNp nload-0.7.2/src/main.cpp nload-0.7.2-new/src/main.cpp
--- nload-0.7.2/src/main.cpp	2008-06-28 11:42:04.000000000 +0200
+++ nload-0.7.2-new/src/main.cpp	2008-10-11 15:12:12.630283324 +0200
@@ -50,7 +50,7 @@
 
 #include <ctype.h>
 #include <time.h>
-#include <curses.h>
+#include <ncurses/curses.h>
 #include <signal.h>
 #include <string.h>
 #undef clear
diff -burNp nload-0.7.2/src/screen.cpp nload-0.7.2-new/src/screen.cpp
--- nload-0.7.2/src/screen.cpp	2008-02-02 17:14:34.000000000 +0100
+++ nload-0.7.2-new/src/screen.cpp	2008-10-11 15:12:45.029897584 +0200
@@ -17,7 +17,7 @@
 
 #include "screen.h"
 
-#include <curses.h>
+#include <ncurses/curses.h>
 #undef clear
 #undef erase
 #undef refresh
diff -burNp nload-0.7.2/src/window.h nload-0.7.2-new/src/window.h
--- nload-0.7.2/src/window.h	2008-02-02 17:14:34.000000000 +0100
+++ nload-0.7.2-new/src/window.h	2008-10-11 15:12:30.253233452 +0200
@@ -22,7 +22,7 @@
 #include <streambuf>
 #include <string>
 
-#include <curses.h>
+#include <ncurses/curses.h>
 #undef clear
 #undef erase
 #undef refresh
diff -burNp nload-0.7.2/configure.in nload-0.7.2-new/configure.in
--- nload-0.7.2/configure.in	2008-06-28 11:50:24.000000000 +0200
+++ nload-0.7.2-new/configure.in	2008-10-11 15:34:38.093125930 +0200
@@ -75,12 +75,12 @@ AC_CHECK_LIB(form, new_form)
 
 dnl Checks for header files.
 AC_HEADER_STDC
-AC_CHECK_HEADERS([limits.h stdlib.h string.h unistd.h curses.h form.h math.h sys/param.h sys/socket.h sys/time.h sys/ioctl.h arpa/inet.h netinet/in.h])
+AC_CHECK_HEADERS([limits.h stdlib.h string.h unistd.h ncurses/curses.h ncurses/form.h math.h sys/param.h sys/socket.h sys/time.h sys/ioctl.h arpa/inet.h netinet/in.h])
 
-if test "$ac_cv_lib_ncurses_initscr $ac_cv_header_curses_h" != "yes yes"; then
+if test "$ac_cv_lib_ncurses_initscr $ac_cv_header_ncurses_curses_h" != "yes yes"; then
     AC_MSG_ERROR([ncurses library or development files not found. ncurses is required for $PACKAGE_NAME.])
 fi
-if test "$ac_cv_lib_form_new_form $ac_cv_header_form_h" != "yes yes"; then
+if test "$ac_cv_lib_form_new_form $ac_cv_header_ncurses_form_h" != "yes yes"; then
     AC_MSG_ERROR([form library or development files not found. These are part of ncurses which is required for $PACKAGE_NAME.])
 fi
 
